apps/mail/lib/theme-editor.ts

```
@@ -0,0 +1,34 @@
+import type { ThemeStyles } from "./themes";
+
+export interface BaseEditorState {
+    styles: ThemeStyles;
+}
+
+export type EditorControls = {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)